src/ApiPort/CommandLine/AnalyzeOptions.cs

```diff
@@ -112,6 +114,11 @@ public override IEnumerable<string> InvalidInputFiles
 
             private void UpdateRequestFlags(Options options)
             {
+                if (options.ShowRetargettingIssues)
+                {
+                    RequestFlags |= AnalyzeRequestFlags.ShowRetargettingIssues;
```

Comment:
Good point. --u doesn't mean anything with --b; it probably ought to enable ShowBreakingChanges automatically.